native/chat/chat-thread-background.react.js

```diff
@@ -15,6 +20,35 @@ type BackgroundChatThreadListProps = {
 export default function BackgroundChatThreadList(
   props: BackgroundChatThreadListProps,
 ) {
+  const chatThreadListData = useSelector(state => chatListData(state));
+
+  const findUnreadBackgroundThreadsNumber = (
+    chatThreadsListData: Array<ChatThreadItem>,
+  ): number => {
+    return chatThreadsListData
```

Comment:
Let's define this function in `lib`, since we anticipate using it on the `web` side pretty soon.

We can put it next to `unreadCount` in `lib/selectors/thread-selectors.js`, since it's a similar problem. And let's have it depend directly on `state.threadStore.threadInfos` there instead of going through `chatListData`.